start.sh

```diff
@@ -8,6 +8,7 @@ nitriding \
 	-prometheus-port 9090 \
 	-vsock-ext \
 	-disable-keep-alives \
+	-prometheus-namespace "starrandsrv" \
```

Comment:
We don't need to do this any more; the `job` attribute is sufficient to distinguish the star-randsrv deployment. Please leave this off unless you specifically need it for a different dashboard.